libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -53,6 +59,12 @@ void ExtVotesPacketHandler::sendPbftNextVotes(dev::p2p::NodeID const &peer_id,
       }
```

Comment:
^^^ you are marking all votes as known before they are send

Reply:
Fixed